src/NServiceBus.Persistence.CosmosDB.AzureStorageSagaExporter.Asp3.AcceptanceTests/NServiceBusAcceptanceTest.cs

```diff
@@ -0,0 +1,94 @@
+﻿namespace NServiceBus.Persistence.CosmosDB.AzureStorageSagaExporter.AcceptanceTests
+{
+    using System;
+    using System.IO;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Cosmos;
+    using Microsoft.Azure.Cosmos.Fluent;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public abstract class NServiceBusAcceptanceTest
+    {
+        [SetUp]
+        public virtual Task SetUp()
+        {
+            AcceptanceTesting.Customization.Conventions.EndpointNamingConvention = t =>
+            {
+                var classAndEndpoint = t.FullName.Split('.').Last();
+
+                var testName = classAndEndpoint.Split('+').First();
+
+                testName = testName.Replace("When_", "");
+
+                var endpointBuilder = classAndEndpoint.Split('+').Last();
+
+
+                testName = Thread.CurrentThread.CurrentCulture.TextInfo.ToTitleCase(testName);
+
+                testName = testName.Replace("_", "");
+
+                return testName + "." + endpointBuilder;
+            };
+
+            return Task.CompletedTask;
+        }
+
+        [TearDown]
+        public virtual Task TearDown()
+        {
+            return Task.CompletedTask;
+        }
+
+        [OneTimeSetUp]
+        public async Task OneTimeSetUp()
+        {
+            var connectionStringEnvironmentVariableName = "CosmosDBPersistence_ConnectionString";
+            var connectionString = GetEnvironmentVariable(connectionStringEnvironmentVariableName,
+                fallbackEmulatorConnectionString: "AccountEndpoint=https://localhost:8081/;AccountKey=C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==");
```

Comment:
Should we merge that connection string, it points to the emulator, but the account key won't make sense on another machine, right?

Reply:
It does make sense. It is universal as long as you don't switch the AccountKey which is a custom setup step